config.json

```
@@ -12,13 +12,23 @@
   "version": 3,
   "online_editor": {
     "indent_style": "space",
-    "indent_size": 2
+    "indent_size": 2,
+    "ace_editor_language": "delphi",
+    "highlightjs_language": "delphi"
```

```suggestion
    "highlightjs_language": "pascal_highlight_rules"
```
I don't know if I should put `pascal` inside the quotes or go with my suggested change.

---

I think it should be pascal:

```suggestion
    "highlightjs_language": "pascal"
```